https://bugs.gentoo.org/390805

fix by Egor Y. Egorov for building with newer libftdi versions

--- a/ftdi_eeprom/main.c
+++ b/ftdi_eeprom/main.c
@@ -106,7 +106,7 @@ int main(int argc, char *argv[]) {
 
     eeprom.vendor_id = cfg_getint(cfg, "vendor_id");
     eeprom.product_id = cfg_getint(cfg, "product_id");
-    eeprom.BM_type_chip = cfg_getbool(cfg, "BM_type_chip");
+    eeprom.chip_type = cfg_getbool(cfg, "BM_type_chip");
 
     eeprom.self_powered = cfg_getbool(cfg, "self_powered");
     eeprom.remote_wakeup = cfg_getbool(cfg, "remote_wakeup");
